File: SofiForce.Models/Models/EntityModels/PromotionItemBundleModel.cs
using System;

namespace Models
{
    public class PromotionItemBundleModel
    {
		public Int32? BundleId { get; set; }
		public Int32? PromotionId { get; set; }
		public int ItemId { get; set; }
		public string ItemCode { get; set; }

		public Int32? Quantity { get; set; }
		public bool? IsMandatory { get; set; }
		public Int32? CBy { get; set; }
		public DateTime? CDate { get; set; }
		public Int32? EBy { get; set; }
		public DateTime? EDate { get; set; }
	}
}

File: SofiForce.Web/ClientApp/src/app/core/Models/EntityModels/PromotionItemBundleModel.ts
export class PromotionItemBundleModel {
    bundleId: number | null;
    promotionId: number | null;
    itemId: number;
    itemCode: string;

    quantity: number | null;
    isMandatory: boolean | null;
    cBy: number | null;
    cDate: Date | null;
    eBy: number | null;
    eDate: Date | null;
}